.github/workflows/deploy_to_s3_prod.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will build the static files
+# and upload them to the given s3 bucket
+
+on:
+  push:
+    branches: ["release"]
```

Comment:
This was in different PR also, no? Merge into this one so that only relevant changed files can be shown

Reply:
Yes, was waiting for that PR to be reviewed. Have merged it in now!

Reply:
Hmm.. understood. Then, this one should be branch off of that one, or this one shouldn't contain that change at all. PRs should be as atomic as possible